ansible/roles/deploy-contrail-with-cad/tasks/main.yml

```diff
@@ -0,0 +1,29 @@
+---
+
+- name: Install build dependencies
+  package:
+    name: "{{ item }}"
+    state: present
+  become: true
+  with_items:
```

Comment:
Please install this requirement with pip (https://github.com/Juniper/contrail-windows-ci/blob/master/ansible/roles/contrail-ansible-deployer/tasks/main.yml)